src/Mod/Arch/ArchMaterial.py

```diff
@@ -21,7 +21,7 @@
 
 import FreeCAD
 if FreeCAD.GuiUp:
-    import FreeCADGui, Arch_rc, os
```

Comment:
Baware that importing the _rc file is needed to load the resources and be able to use any of the :/icons/ later on, in this case at L206:
`return ":/icons/Arch_Material_Group.svg"`